src/monster.cpp

```diff
@@ -1575,8 +1575,9 @@ bool monster::move_effects( bool )
 void monster::add_effect( const efftype_id &eff_id, const time_duration dur, body_part bp,
                           bool permanent, int intensity, bool force, bool deferred )
 {
-    bp = num_bp; // Effects are not applied to specific monster body part
-    Creature::add_effect( eff_id, dur, bp, permanent, intensity, force, deferred );
+    // Effects are not applied to specific monster body part
+    ( void ) bp;
```

Comment:
Would it be better to just comment out the parameter name? i.e.
`monster::add_effect( ..., body_part /*bp*/ ... )`